src/components/Dictionary/DictionaryContainer.js

```diff
@@ -4,40 +4,25 @@ import PropTypes from 'prop-types'
 // FPCC
 import DictionaryPresentation from 'components/Dictionary/DictionaryPresentation'
 import DictionaryData from 'components/Dictionary/DictionaryData'
-import Loading from 'components/Loading'
 import { TYPE_DICTIONARY } from 'common/constants'
 import SiteDocHead from 'components/SiteDocHead'
 
 function DictionaryContainer({ searchType = TYPE_DICTIONARY, kids = null }) {
-  const {
-    actions,
-    infiniteScroll,
-    isLoading,
-    isLoadingEntries,
-    items,
-    labels,
-    moreActions,
-    sitename,
-    loadRef,
-    count,
-  } = DictionaryData({ searchType, kids })
+  const { infiniteQueryResponse, labels, sitename } = DictionaryData({
+    searchType,
+    kids,
+  })
   return (
-    <Loading.Container isLoading={isLoading}>
+    <>
```

Comment:
Just curious, are these empty blocks necessary? I noticed them in a lot of the container files.

Reply:
The empty blocks are react Fragments. JSX elements require a single parent element, and using a div can impact styling so sometimes a fragment is used instead.